test/studies/shootout/submitted/spectralnorm.chpl

```diff
@@ -38,7 +38,7 @@ proc multiplyAv(v: [?Dv], Av: [?DAv]) {
 }
 
 //
-// Compute A-transpose * v ('Atv').
+// Compute A-tranpose * v ('Atv').
```

Comment:
Shouldn't this just stay as "transpose"? Or are we being very stringent about the sync?